queenbee/cli/operator/new.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from ...base.parser import parse_file
+from ...operator import Operator
+
+try:
+    import click
+    from click_plugins import with_plugins
+except ImportError:
+    raise ImportError(
+        'click modules not installed. Try `pip install queenbee[cli]` command.'
+    )
+
+MODULE_PATH = os.path.abspath(os.path.dirname(__file__))
+
+@click.command('new')
+@click.argument('name')
+@click.option(
+    '-p',
+    '--path',
+    help='The path at which to create the new operator'
+)
```

Comment:
we should probably clarify what will be the default value if path is not provided.